tests/Unit/Numerical/LinearAlgebra/Test_Determinant.cpp

```diff
@@ -0,0 +1,137 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include <catch.hpp>
+
+#include "Numerical/LinearAlgebra/Determinant.hpp"
+
+TEST_CASE("Unit.Numerical.LinearAlgebra.Determinant",
+          "[LinearAlgebra][Numerical][Unit]") {
+  // Test determinant function on general (no symmetry) matrices:
+  // * Use rank-2 Tensor in 1-4 dimensions (i.e. 1x1, 2x2, 3x3, 4x4 matrices).
+  // * Use length-1 DataVector (i.e. data at single spatial point)
+  SECTION("Test general matrices") {
+    const size_t n_pts = 1;
+
+    // Test 1D matrix
+    {
+      tnsr::ij<DataVector, 1, Frame::Grid> matrix1D(n_pts, 1.3);
+      auto det1D = determinant(matrix1D);
+      CHECK(1.3 == Approx(det1D.get()[0]));
```

Comment:
Should be setting a tighter tolerance for Approx. See the BitBucket tests for wedge maps or the Catch documentation.